eksperymenty_programistyczne_na_edoku4.rst

```diff
@@ -0,0 +1,91 @@
+Eksperymenty programistyczne na eDoku4
+======================================
+
+Pierwszy eksperyment programistyczny
+------------------------------------
+
+Cel eksperymentu
+~~~~~~~~~~~~~~~~
+
+Celem eksperymentu jest wstępne wysondowanie możliwości rozbudowy testowanego systemu. Jego wynik może pomóc w opracowaniu kolejnych, bardziej wyszukanych eksperymentów.
+
+Założenia eksperymentu
+~~~~~~~~~~~~~~~~~~~~~~
+
+Instrukcja kancelaryjna w ramach minimalnego zestawu metadanych opisujących przesyłki wpływające oraz wychodzące wymienia (opcjonalny) adres email odpowiednio nadawcy oraz adresata. Urząd mógłby chcieć wdrożyć narzędzie umożliwiające Obywatelom zalogowanie się swoim adresem email oraz wyświetlenie przesyłek powiązanych ze swoim kontem. Eksperyment ma za zadanie ustalić możliwość łatwego napisania takiej aplikacji.
+
+Forma eksperymentu
+~~~~~~~~~~~~~~~~~~
+
+W ramach eksperymentu zbadana zostanie możliwość wydobycia z bazy danych poszczególnych systemów EZD metadanych przesyłek powiązanych z konkretnym adresem email. Pierwszoplanową metodą osiągnięcia tego celu będzie skorzystanie z API; w dalszej kolejności rozpatrywane będą inne metody, o ile nie będą się one wiązały z nadmierną uciążliwością programistyczną. Jeżeli uzyskanie metadanych przesyłek okaże się możliwe, zbadana zostanie także możliwość uzyskania odwzorowań cyfrowych tych przesyłek.
```

Comment:
Dyskusja o tym czy pisać 'email' czy 'e-mail' nie pojawia się pierwszy raz. Warto żeby w tym dokumencie nazewnictwo było spójne (nie jest).

Poprawię to później, bo jeszcze nie jestem pewien "w którą stronę".